motionplan/plan.go

```diff
@@ -334,13 +334,18 @@ func CalculateFrameErrorState(e ExecutionState, frame referenceframe.Frame) (spa
 	if index < 0 || index >= len(path) {
 		return nil, fmt.Errorf("index %d out of bounds for Path of length %d", index, len(path))
 	}
-	pose, ok := path[index][frame.Name()]
+	pose, ok := path[index][executionFrame.Name()]
```

Comment:
I dont think this is correct the path should be for planning frames no?

Reply:
the planningFrame is the same as the executionFrame in terms of their name and what they are under the hood.

Reply:
I prefer to use the name `executionFrame` instead of `planningFrame`

Reply:
I think this is confusing and also prone to breaking should those names should ever change

Reply:
What names?